consumers/notifiers/service_test.go

```diff
@@ -74,20 +74,24 @@ func TestCreateSubscription(t *testing.T) {
 
 	for _, tc := range cases {
 		repoCall := auth.On("Identify", mock.Anything, &magistrala.IdentityReq{Token: tc.token}).Return(&magistrala.IdentityRes{Id: testsutil.GenerateUUID(t)}, nil)
+		repoCall1 := repo.On("Save", mock.Anything, mock.Anything).Return(tc.id, tc.err)
 		id, err := svc.CreateSubscription(context.Background(), tc.token, tc.sub)
 		assert.True(t, errors.Contains(err, tc.err), fmt.Sprintf("%s: expected %s got %s\n", tc.desc, tc.err, err))
 		assert.Equal(t, tc.id, id, fmt.Sprintf("%s: expected %s got %s\n", tc.desc, tc.id, id))
 		repoCall.Unset()
+		repoCall1.Unset()
 	}
 }
 
 func TestViewSubscription(t *testing.T) {
-	svc, auth := newService()
+	svc, auth, repo := newService()
 	sub := notifiers.Subscription{Contact: exampleUser1, Topic: "valid.topic"}
 	repoCall := auth.On("Identify", mock.Anything, &magistrala.IdentityReq{Token: exampleUser1}).Return(&magistrala.IdentityRes{Id: validID}, nil)
+	repoCall1 := repo.On("Save", mock.Anything, mock.Anything).Return(validID, nil)
```

Comment:
No need for `CreateSubscription` before running test cases since we are not persisting that data